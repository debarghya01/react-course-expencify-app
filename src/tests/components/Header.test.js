import React from 'react';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import Header from '../../components/Header';

test('testing header component', () => {
    const wrapper = shallow(<Header />)
    // expect(wrapper.find('h1').text()).toBe('Expensify !!!!!')
    // expect(toJSON(wrapper)).toMatchSnapshot()
    expect(wrapper).toMatchSnapshot()

    // const renderer = new ReactShallowRenderer()
    // renderer.render(<Header />)
    // expect(renderer.getRenderOutput()).toMatchSnapshot()
})