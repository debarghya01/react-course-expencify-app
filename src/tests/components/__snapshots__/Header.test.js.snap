// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing header component 1`] = `
<header>
  <h1>
    Expensify !!!!!
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    Help page
  </NavLink>
</header>
`;
